script/options.js

```diff
@@ -18,6 +18,11 @@ const COMMAND_LINE_OPTIONS_DEFINITIONS = [
   { name: 'protocol', type: String, defaultValue: 'http' },
   { name: 'public', type: String, defaultValue: null },
   { name: 'destination', type: String, defaultValue: null },
+  {
+    name: 'content-directory',
+    type: String,
+    defaultValue: '../../vagov-content/pages',
+  },
```

Comment:
I added this parameter because a repo's directory name is dynamic on Heroku. Because of this, I had to manually supply the directory name to this build script from the `vagov-content` review instance setup.